src/components/CarouselForCertifications/CarouselCertifications.vue

```diff
@@ -50,8 +50,8 @@ onMounted(async () => {
   const { loadCertifications } = await runQuery(LoadCertifications, {
     title: "iso",
   });
-  if (Array.isArray(getCertifications)) {
-    certifications.value = getCertifications.map((certification: any) => {
+  if (Array.isArray(loadCertifications)) {
+    certifications.value = loadCertifications.map((certification: unknown) => {
```

Comment:
Em vez de usar o `as Certification` faz assim: `loadCertifications.map((certification: unknown): Certification` evita forçar tipagem usando o as